### PR TITLE
Web tier fat updates

### LINKED ISSUE #27257: Web Tier FAT updates
1) Update `expressionLanguage` in `tested.features` to `expressionlanguage`.
2) Use the new `OR_LATER_FEATURES` for our `SkipForRepeat` annotations.

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #27257

1) Update `expressionLanguage` in `tested.features` to `expressionlanguage`.
2) Use the new `OR_LATER_FEATURES` for our `SkipForRepeat` annotations.

### CODE REVIEW

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#build

**Comment by pnicolucci** [comment]
#libby

File: dev/com.ibm.ws.jsf.2.2_fat/bnd.bnd [hunk 1]
```diff
@@ -76,7 +76,7 @@ tested.features: \
     jsf-2.3,\
     cdi-2.0,\
     beanValidation-2.0,\
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.2.2_fat/bnd.bnd]: This could be lowercase, too.

File: dev/com.ibm.ws.jsf.2.3_fat/bnd.bnd [hunk 2]
```diff
@@ -29,7 +29,7 @@ tested.features: \
  xmlBinding-3.0,\
```
Comment (volosied) [inline dev/com.ibm.ws.jsf.2.3_fat/bnd.bnd]: Same here

File: dev/io.openliberty.webcontainer.servlet.6.0.internal_fat/bnd.bnd [hunk 3]
```diff
@@ -30,7 +30,7 @@ fat.project: true
 tested.features:\
     noship-1.0,\
     servlet-6.1,\
-    expressionLanguage-6.0,\
+    expressionlanguage-6.0,\
```
Comment (volosied) [inline dev/io.openliberty.webcontainer.servlet.6.0.internal_fat/bnd.bnd:33]: Copyright or no?  Then again, this is only a one letter change.

Comment (volosied) [reply dev/io.openliberty.webcontainer.servlet.6.0.internal_fat/bnd.bnd:33]: I see you didn't modify the other bnds, so probably no need.

Comment (pnicolucci) [reply dev/io.openliberty.webcontainer.servlet.6.0.internal_fat/bnd.bnd:33]: Yea no copyright update here, no functional change, just fixing a formatting issue is how I'm looking at it.

**Comment by pnicolucci** [comment]
#libby #build

**Review by volosied — APPROVED** [review]

**Pull request merged**